eogrow/pipelines/export_maps.py

```diff
@@ -61,16 +61,71 @@ class Schema(Pipeline.Schema):
     config: Schema
 
     def run_procedure(self) -> Tuple[List[str], List[str]]:
-        """Procedure which downloads satellite data"""
+        """Extracts and merges the data from EOPatches into a TIFF file.
+
+        1. Extracts data from EOPatches via workflow into per-EOPatch tiffs.
+        2. For each UTM zone:
+            - Prepares tiffs for merging (transfers to local if needed).
+            - Merges the tiffs, cogification is done if requested.
+            - The output file is finalized (transferred if needed) and per-EOPatch tiffs are cleaned.
+
+        """
+
         successful, failed = super().run_procedure()
 
-        _, feature_name = self.config.feature
-        self.create_maps(feature_name, successful)
+        if not successful:
+            raise ValueError("Failed to extract tiff files from any of EOPatches.")
```

Comment:
I don't really like that either. Especially since the rest of the pipeline copies files around and takes some time, just for us to get garbage.

If we remove it, should we perhaps do cleanup in case exporting failed? Or do we just leave a bunch of randomly named tiffs?

Reply:
I'm not sure I understand. At this point nothing should be copied yet, right? Only the workflow was run, and if there are any failed patches, then I suggest to just stop there and then. Existing tiffs from the workflow are fine anyway, while others are non-existing, since the workflow failed and one just needs to rerun the workflow for the failed ones until there are none left.

Reply:
the workflow-exported tiffs are given unique names (bound to the unique pipeline id) just in case multiple pipelines are exporting similar things concurrently. So re-running a (now fixed) pipeline won't clear the old tiffs.

Reply:
Discussed in private. My preference is that either of the following happens if there are any failed exported:
- tiff names are made simpler (uniqueness is dropped) and could be left in (this way also the skip existing would work, right?)
- or that the tiff files are simply cleaned up

Exporting doesn't apply any transformations to the eopatches (unless I'm mistaken), so in that respect if something goes wrong, it's likely due to the process failing or something else being wrong already before the pipeline. In such cases one can re-run the whole thing or run the pipeline for a single patch and debug further.